challenges/ecosystem/05.md

```diff
@@ -215,3 +215,39 @@ Now you can run `npm run test:watch` to watch your files and run tests automatic
 
 - https://reactnativetesting.io/
 - https://github.com/vanGalilea/react-native-testing?tab=readme-ov-file#covered-examples-
+
+
+### 👽 Bonus
+
+We can automate the user interaction in the app with [`Maestro`](https://maestro.mobile.dev/) —a library that allows you to write end-to-end tests for your React Native app.
```

Comment:
keep the bonus section here with a link to the cookbook https://callstack.github.io/react-native-testing-library/cookbook/index